HISTORY.md

```diff
@@ -1,4 +1,32 @@
-# Release Notes
+# Release N## v0.14.0 - 2024-06-13
```

Comment:
```suggestion
# Release N## v0.14.0 - 2024-06-13
```
```suggestion
# v0.14.0 - 2024-06-13
```

Reply:
How did this happen?

Reply:
I think it got clipped due to the [release notes script](https://github.com/sdv-dev/SDV/blob/main/scripts/release_notes_generator.py)

This line needs to updated to look for `Release Notes`
https://github.com/sdv-dev/SDV/blob/main/scripts/release_notes_generator.py#L136